lib/rubocop/rspec/language.rb

```diff
@@ -40,6 +40,10 @@ def send_pattern
           "(send #{RSPEC} #{node_pattern_union} ...)"
         end
 
+        def send_or_block_or_block_pass_pattern
```

Comment:
I don't like where this is going, but currently, we do not have a better solution